# Links to software collection pages

- [FriendsDontLetFriends GITHUB help page for visualizations](https://github.com/cxli233/FriendsDontLetFriends)
- [Pop Gen Software List](http://www.duckdna.org/softwares/)
- [Database of Bioinformatics Software Tools and Resources](https://bioinformaticshome.com/db/)
- [Courses.washington Software](https://courses.washington.edu/popgen/Software.htm)
- [Bioinformaticshome SNP-TOOLS](https://bioinformaticshome.com/tools/SNP-tools.html)
- [zsl Softwares suggestions](https://www.zsl.org/about-zsl/resources/software)
- [PopGen Simulation Framework Slim](https://messerlab.org/slim/) [Slim](https://www.slimframework.com/docs/v3/)
- [Manhattanplot in R](https://r-graph-gallery.com/101_Manhattan_plot.html)
- [Relationship inference with KING](https://www.kingrelatedness.com/manual.shtml)
- [GnomAD harmonizing human exome and genome data](https://gnomad.broadinstitute.org/)
- [Getting Things Done resources and code blog](https://gettinggeneticsdone.blogspot.com/)
- [Integrated pipeline for analysis and vis.of pop. structure and relatedness based on genome-wide data](https://bio.tools/psrelip)
- [Visualize Plink Pedigree files](https://bioinformatics.stackexchange.com/questions/4903/visualize-plink-pedigree-files)
- [QC workflow by hbctraining](https://hbctraining.github.io/DGE_workshop/lessons/03_DGE_QC_analysis.html)
- [Workflow for merging bcf/vcf files](https://www.biocomputix.com/post/how-to-combine-merge-vcf-bcf-files-using-bcftools-merge)
- [GATK phred quality scores info](https://gatk.broadinstitute.org/hc/en-us/articles/360035531872-Phred-scaled-quality-scores)
- [Scripts to simulate and estimate effect of reduced genomic representation(with article link)](https://github.com/EluLava/ROHsReducedRep/tree/main)
- [Virtual population genomics website](https://virtualbiologylab.org/population-genetics/)
- [Population genomics simulations](https://gdc-docs.ethz.ch/UniBS/HS2020/PopGen/site/PopG_Simulation/)
- [European nucleotide archive](https://www.ebi.ac.uk/ena/browser/home)
- [European Variation Archive](https://www.ebi.ac.uk/eva/?RS-Release&releaseVersion=5)
- [EVA rsID](https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_5/by_species/)
- [Awesome genome visualizations](https://cmdcolin.github.io/awesome-genome-visualization/?latest=true)
- [R karyoplotR package](https://bernatgel.github.io/karyoploter_tutorial/)
- [NAGRP Data Repository](https://www.animalgenome.org/repository/)
- [Animal Genome Database](https://www.animalgenome.org/repository/)
- [NCBI](https://www.ncbi.nlm.nih.gov/)
- [Ensembl](https://www.ensembl.org/index.html)
- [OMIA](https://omia.org/home/)
- [OMIM](https://www.omim.org/)
- [Orthodb](https://www.orthodb.org/)
- [Genecards](https://www.genecards.org/)
- [EVA program](https://www.nordgen.org/en/our-work/nordgen-farm-animals/eva-program/)
- [How to execute GWAS guide](https://github.com/ucdavis-bioinformatics-training/2021-July-Genome-Wide-Association-Studies) [visualization](https://ucdavis-bioinformatics-training.github.io/2021-July-Genome-Wide-Association-Studies/data_analysis/GWASvisualizations.html) [workshop](https://ucdavis-bioinformatics-training.github.io/2021-July-Genome-Wide-Association-Studies/data_analysis/Plink_Stepbystep)
- [Equine SNP database](http://webtom.cabgrid.res.in/eqsnpdb/index.php)
- [Ways to measure genetic diversity](http://agtr.ilri.cgiar.org/quantitative-methods/measuring-genetic-diversity-molecular-data#:~:text=There%20are%20two%20commonly%20used,diversity%20within%20and%20between%20populations.&text=The%20decline%20in%20heterozygosity%20due,known%20as%20the%20fixation%20index.)
- [KEGG pathways](https://www.genome.jp/kegg/pathway.html)
- [SRPlot](https://www.bioinformatics.com.cn/srplot)
- [Sequence ontology browser](http://www.sequenceontology.org/miso/current_svn/term/SO:0001583)
- [Sci-hub](https://sci-hub.se/)
- [PhyloP](http://compgen.cshl.edu/phast/phyloP-tutorial.php#bcl)
- [SHAPEIT](https://odelaneau.github.io/shapeit5/)
- [Markov and hidden Markov models](https://towardsdatascience.com/markov-and-hidden-markov-model-3eec42298d75)
- [Markov chain monte carlo probabilities](https://machinelearningmastery.com/markov-chain-monte-carlo-for-probability/)
- [Understanding pbwt](http://alimanfoo.github.io/2016/06/21/understanding-pbwt.html)
- [Computational Genomics with R](https://compgenomr.github.io/book/)
- [FAO-DAD](https://www.fao.org/dad-is/browse-by-country-and-species/en/)
- [Equine recombination Map](https://osf.io/e72ah/)
- [GCTA](https://yanglab.westlake.edu.cn/software/gcta/#GREML)
- [findHap](https://www.ars.usda.gov/northeast-area/beltsville-md-barc/beltsville-agricultural-research-center/agil/aip/software/findhap/)
- [Cdrom2021](https://ibg.colorado.edu/cdrom2021/)
- [Technical foundations of Informatics](https://info201.github.io/index.html)
- [Impute5](https://jmarchini.org/software/#impute-5)
- [Collection of Software by jmarchini](https://jmarchini.org/software/)
- [Genomics Boot Camp](https://genomicsbootcamp.github.io/book/)
- [PLINK](https://www.cog-genomics.org/plink/2.0/)
- [SAGE](http://darwin.cwru.edu/sage/pages/system_requirement.php)
- [Bioinformatics tutorials](https://guides.utmb.edu/c.php?g=411466&p=2804096)
- [NCBI Genome Workbench](https://www.ncbi.nlm.nih.gov/tools/gbench/)
- [NCBI Selected Analysis tools](https://www.ncbi.nlm.nih.gov/home/analyze/)
- [Galaxy](https://usegalaxy.org/)
- [Some Norwegian Equine Pedigree data](https://sportsearch.travsport.no/Pages/?modus=2)
- [Some Norwegian Equine test matings](https://sportords.travsport.no/ords/f?p=135:1:10096673589726:::::)
- [bcftools](https://samtools.github.io/bcftools/bcftools.html)

# Published Lists of Pop Gen Softwares by duckdna.com
- [Excoffier & Heckel (2006) Computer programs for population genetics data analysis: a survival guide. Nature Reviews Genetics 7:745-758.](http://www.readcube.com/articles/10.1038/nrg1904)
- [Liu, Athanasiadis, & Weale (2008) A survey of genetic simulation software for population and epidemiological studies. Human Genomics 3:79-86.](https://humgenomics.biomedcentral.com/articles/10.1186/1479-7364-3-1-79)
