# Software Collection Pages

## Table of Contents
- [Population Genetics and Genomics](#population-genetics-and-genomics)
- [Bioinformatics](#bioinformatics)
- [Genomic Data Analysis](#genomic-data-analysis)
- [Genomic Databases and Repositories](#genomic-databases-and-repositories)
- [Genomic Visualization Tools](#genomic-visualization-tools)
- [Other Resources](#other-resources)


## Population Genetics and Genomics

1. **[FriendsDontLetFriends GITHUB help page for visualizations](https://github.com/cxli233/FriendsDontLetFriends)**: A GitHub repository that provides resources for creating data visualizations.

2. **[Pop Gen Software List](http://www.duckdna.org/softwares/)**: A collection of software tools for population genetics research.

3. **[Courses.washington Software](https://courses.washington.edu/popgen/Software.htm)**: A list of software tools used in the population genetics course at the University of Washington.

4. **[PopGen Simulation Framework Slim](https://messerlab.org/slim/)**: A flexible, forward-time population genetics simulation framework.

5. **[Virtual population genomics website](https://virtualbiologylab.org/population-genetics/)**: An online platform for learning and practicing population genetics simulations.

6. **[Population genomics simulations](https://gdc-docs.ethz.ch/UniBS/HS2020/PopGen/site/PopG_Simulation/)**: A resource for population genomics simulations, provided by ETH Zurich.

7. **[Slim](https://www.slimframework.com/docs/v3/)**: A PHP micro framework that helps you quickly write simple yet powerful web applications and APIs.

8. **[Progenygenetics](https://www.progenygenetics.com/online-pedigree/)**: Online tool for drawing pedigrees.

9. **[Eff. Popsize Softwares](https://people.sc.fsu.edu/~pbeerli/popgensoftware.html)**: Different tools for estimating Ne

## Bioinformatics

1. **[Database of Bioinformatics Software Tools and Resources](https://bioinformaticshome.com/db/)**: A comprehensive database of bioinformatics software tools and resources.

2. **[Bioinformaticshome SNP-TOOLS](https://bioinformaticshome.com/tools/SNP-tools.html)**: A collection of tools for SNP (Single Nucleotide Polymorphism) analysis in bioinformatics.

3. **[zsl Softwares suggestions](https://www.zsl.org/about-zsl/resources/software)**: A list of software suggestions from the Zoological Society of London.

4. **[Relationship inference with KING](https://www.kingrelatedness.com/manual.shtml)**: A tool for inferring relatedness from large scale genotyping data.

5. **[Scripts to simulate and estimate effect of reduced genomic representation(with article link)](https://github.com/EluLava/ROHsReducedRep/tree/main)**: A GitHub repository containing scripts to simulate and estimate the effect of reduced genomic representation.

6. **[SHAPEIT](https://odelaneau.github.io/shapeit5/)**: A fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data.

7. **[findHap](https://www.ars.usda.gov/northeast-area/beltsville-md-barc/beltsville-agricultural-research-center/agil/aip/software/findhap/)**: A software for haplotype reconstruction and estimation of missing genotypes.

8. **[Impute5](https://jmarchini.org/software/#impute-5)**: A software for genotype imputation.

9. **[Collection of Software by jmarchini](https://jmarchini.org/software/)**: A collection of software developed by Jonathan Marchini.

10. **[PLINK](https://www.cog-genomics.org/plink/2.0/)**: A free, open-source whole genome association analysis toolset, designed to perform a range of basic, large-scale analyses in a computationally efficient manner.

11. **[SAGE](http://darwin.cwru.edu/sage/pages/system_requirement.php)**: Statistical Analysis for Genetic Epidemiology, a free software package containing programs for use in the genetic analysis of family, pedigree and population data.

12. **[bcftools](https://samtools.github.io/bcftools/bcftools.html)**: A set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF.
    
13. **[SnpEFF and SnpSift softwear](http://pcingola.github.io/SnpEff/snpeff/introduction/)**: SnpEFF predicts the impact of an alteration on genes (such as missense, nonsense, etc.), while SnpSift helps in filtering and manipulating genomic annotated variants.

14. **[phasedibd](https://github.com/23andMe/phasedibd/tree/master)**: This is a software tool for detecting identity by descent (IBD) segments in phased genotype data. IBD detection is useful in genetic studies for identifying related individuals, mapping disease genes, and inferring population history.

15. **[Selscan](https://github.com/szpiech/selscan/tree/master)**: This is a software tool designed to scan for positive selection in genomic data. It implements the Extended Haplotype Homozygosity (EHH), Integrated Haplotype Score (iHS), and Cross Population Extended Haplotype Homozygosity (XP-EHH) tests, among others.

## Genomic Data Analysis

1. **[GnomAD harmonizing human exome and genome data](https://gnomad.broadinstitute.org/)**: A resource for exploring human genomic diversity.

2. **[Integrated pipeline for analysis and vis.of pop. structure and relatedness based on genome-wide data](https://bio.tools/psrelip)**: A tool for analyzing population structure and relatedness based on genome-wide data.

3. **[Visualize Plink Pedigree files](https://bioinformatics.stackexchange.com/questions/4903/visualize-plink-pedigree-files)**: A StackExchange thread discussing methods to visualize Plink Pedigree files.

4. **[QC workflow by hbctraining](https://hbctraining.github.io/DGE_workshop/lessons/03_DGE_QC_analysis.html)**: A quality control workflow for Differential Gene Expression analysis, provided by hbctraining

5. **[Workflow for merging bcf/vcf files](https://www.biocomputix.com/post/how-to-combine-merge-vcf-bcf-files-using-bcftools-merge)**: A guide for merging BCF/VCF files using bcftools.

6. **[GATK phred quality scores info](https://gatk.broadinstitute.org/hc/en-us/articles/360035531872-Phred-scaled-quality-scores)**: Information on Phred-scaled quality scores in the Genome Analysis Toolkit (GATK).

7. **[How to execute GWAS guide](https://github.com/ucdavis-bioinformatics-training/2021-July-Genome-Wide-Association-Studies)**: A guide on how to execute Genome-Wide Association Studies (GWAS).

8. **[Ways to measure genetic diversity](http://agtr.ilri.cgiar.org/quantitative-methods/measuring-genetic-diversity-molecular-data#:~:text=There%20are%20two%20commonly%20used,diversity%20within%20and%20between%20populations.&text=The%20decline%20in%20heterozygosity%20due,known%20as%20the%20fixation%20index.)**: A resource explaining different ways to measure genetic diversity.

9. **[Markov and hidden Markov models](https://towardsdatascience.com/markov-and-hidden-markov-model-3eec42298d75)**: An article explaining the concept of Markov and Hidden Markov models.

10. **[Markov chain monte carlo probabilities](https://machinelearningmastery.com/markov-chain-monte-carlo-for-probability/)**: A guide on how to use Markov Chain Monte Carlo for probability.

11. **[Understanding pbwt](http://alimanfoo.github.io/2016/06/21/understanding-pbwt.html)**: A blog post explaining the concept of Positional Burrows-Wheeler Transform (PBWT).

12. **[Computational Genomics with R](https://compgenomr.github.io/book/)**: An online book on computational genomics with R.

13. **[GCTA](https://yanglab.westlake.edu.cn/software/gcta/#GREML)**: A software tool for Genome-wide Complex Trait Analysis.

14. **[Cdrom2021](https://ibg.colorado.edu/cdrom2021/)**: A resource for the 2021 CD-ROM of the Behavior Genetics Association.

15. **[Genomics Boot Camp](https://genomicsbootcamp.github.io/book/)**: An online book providing a boot camp style introduction to genomics.

16. **[Center for Statistical Genetics Software](https://sph.umich.edu/csg/software.html)**: An overview of softwares for Statistical Genetics.

17. **[Laboratories Statistical Genetics and Genetic Epidemiology: Daniel J. Schaid](https://www.mayo.edu/research/labs/statistical-genetics-genetic-epidemiology/software)**: Overview of software for Statistical Genetic and Epidemiology.

18. **[IPGRI and Cornell University, 2003, Genetic diversity analysis with molecular marker data: Learning module](https://cropgenebank.sgrp.cgiar.org/images/file/learning_space/molecular_markers/volume2/12_Software%2520programs.pdf)**: PPT overview of softwares for investigating genetic diversity using molecular markers.

## Genomic Databases and Repositories

1. **[European nucleotide archive](https://www.ebi.ac.uk/ena/browser/home)**: A comprehensive database of nucleotide sequences and related information.

2. **[European Variation Archive](https://www.ebi.ac.uk/eva/?RS-Release&releaseVersion=5)**: A database of all types of genetic variation data from all species.

3. **[EVA rsID](https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_5/by_species/)**: A resource from the European Variation Archive providing rsID releases.

4. **[Animal Genome Database](https://www.animalgenome.org/repository/)**: A database of animal genomes.

5. **[NCBI](https://www.ncbi.nlm.nih.gov/)**: The National Center for Biotechnology Information, a comprehensive resource for access to biomedical and genomic information.

6. **[Ensembl](https://www.ensembl.org/index.html)**: A genome browser for vertebrate genomes that provides data about genes, proteins and more.

7. **[OMIA](https://omia.org/home/)**: The Online Mendelian Inheritance in Animals (OMIA) database, a catalogue of inherited disorders, other (single-locus) traits, and genes in 251 animal species.

8. **[OMIM](https://www.omim.org/)**: Online Mendelian Inheritance in Man (OMIM), a comprehensive, authoritative compendium of human genes and genetic phenotypes.

9. **[Orthodb](https://www.orthodb.org/)**: The hierarchical catalog of orthologs, a resource for exploring gene evolutionary history.

10. **[Genecards](https://www.genecards.org/)**: A searchable, integrative database that provides comprehensive, user-friendly

11. **[EVA program](https://www.nordgen.org/en/our-work/nordgen-farm-animals/eva-program/)**: The European Varietas Archive (EVA) program, a project for the conservation of genetic resources in Nordic farm animals.

12. **[Equine SNP database](http://webtom.cabgrid.res.in/eqsnpdb/index.php)**: A database dedicated to equine Single Nucleotide Polymorphisms (SNPs).

13. **[FAO-DAD](https://www.fao.org/dad-is/browse-by-country-and-species/en/)**: The Domestic Animal Diversity Information System (DAD-IS) by the Food and Agriculture Organization of the United Nations, providing information on breeds of livestock and poultry.

14. **[Equine recombination Map](https://osf.io/e72ah/)**: A resource providing an equine recombination map.

15. **[NAGRP Data Repository](https://www.animalgenome.org/repository/)**: The National Animal Genome Research Program's data repository, providing a wide range of animal genomics data.

16. **[Some Norwegian Equine Pedigree data](https://sportsearch.travsport.no/Pages/?modus=2)**: A resource providing Norwegian equine pedigree data.

17. **[Some Norwegian Equine test matings](https://sportords.travsport.no/ords/f?p=135:1:10096673589726:::::)**: A resource providing data on Norwegian equine test matings.


## Genomic Visualization Tools

1. **[Manhattanplot in R](https://r-graph-gallery.com/101_Manhattan_plot.html)**: A guide for creating Manhattan plots in R.

2. **[Awesome genome visualizations](https://cmdcolin.github.io/awesome-genome-visualization/?latest=true)**: A collection of tools and resources for visualizing genomic data.

3. **[R karyoplotR package](https://bernatgel.github.io/karyoploter_tutorial/)**: A tutorial for the karyoplotR package in R, used for creating karyoplots.

4. **[SRPlot](https://www.bioinformatics.com.cn/srplot)**: A tool for creating sequence-related plots.

5. **[KEGG pathways](https://www.genome.jp/kegg/pathway.html)**: The Kyoto Encyclopedia of Genes and Genomes (KEGG) PATHWAY database, a collection of manually drawn pathway maps representing knowledge on the molecular interaction and reaction networks.

6. **[Sequence ontology browser](http://www.sequenceontology.org/miso/current_svn/term/SO:0001583)**: A resource for browsing sequence ontology terms.

7. **[PhyloP](http://compgen.cshl.edu/phast/phyloP-tutorial.php#bcl)**: A tool for identifying and visualizing evolutionary conservation using phylogenetic p-values.
  
8. **[GWAS visualization](https://ucdavis-bioinformatics-training.github.io/2021-July-Genome-Wide-Association-Studies/data_analysis/GWASvisualizations.html)**: A guide on how to visualize Genome-Wide Association Studies (GWAS) data.

9. **[Plink_Stepbystep](https://ucdavis-bioinformatics-training.github.io/2021-July-Genome-Wide-Association-Studies/data_analysis/Plink_Stepbystep)**: A step-by-step guide on how to use PLINK, a free, open-source whole genome association analysis toolset.


## Other Resources

1. **[Getting Things Done resources and code blog](https://gettinggeneticsdone.blogspot.com/)**: A blog with resources and code snippets for genetics research.

2. **[Technical foundations of Informatics](https://info201.github.io/index.html)**: A course on the technical foundations of informatics.

3. **[Bioinformatics tutorials](https://guides.utmb.edu/c.php?g=411466&p=2804096)**: A collection of tutorials on various topics in bioinformatics.

4. **[NCBI Genome Workbench](https://www.ncbi.nlm.nih.gov/tools/gbench/)**: A suite of tools provided by NCBI for manipulating and analyzing genomic data.

5. **[NCBI Selected Analysis tools](https://www.ncbi.nlm.nih.gov/home/analyze/)**: A selection of analysis tools provided by NCBI for genomic data.

6. **[Galaxy](https://usegalaxy.org/)**: An open-source platform for accessible, reproducible, and transparent computational research.

7. **[Sci-hub](https://sci-hub.se/)**: A website that provides free access to millions of research papers and books, bypassing publisher paywalls.

8. **[Speciation Genomics](https://speciationgenomics.github.io/haplotypes/)**: Github site with easy and understandable explenations, (rehh package for example).

9. **[Another list of resources](https://www.amnh.org/research/staff-directory/sergios-orestis-kolokotronis/evolutionary-genetics-software)**: American Museum of Natural History links to genetics software, apparently. 


## Citing and grammar help
1. **[Grammarly]**:
2. **[Mendeley]**:
3. something for checking plagiarism percentage.

